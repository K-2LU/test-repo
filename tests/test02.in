10
1262 997 1862 216 5223 1579 4232 3428 603 9645 