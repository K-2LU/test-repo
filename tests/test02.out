4 3 6 1 9 5 8 7 2 10 
